sigle/src/modules/layout/components/DashboardLayout.tsx

```diff
@@ -146,6 +148,9 @@ export const DashboardLayout = ({
     }
   };
 
+  const pathname =
+    typeof window !== 'undefined' && window.location.pathname.split('/');
```

Comment:
This part is not accessible in the `router` object?

Reply:
As this should be shown only on the current user profile this could work no?

Reply:
Ahh yes very true, will update it 👍